src/xmipp/bindings/matlab/tom_xmipp_rotate.cpp

```diff
@@ -25,6 +25,9 @@
 /*xmipp includes */
 #include "tom_xmipp_helpers.h"
 #include <core/xmipp_image.h>
+#include <core/matrix2d.h>
```

Comment:
why do you add these includes (multiple occurrences) ?

Reply:
Very likely, with the include reorganization to speed up the compilation, the Matlab compilation was broken. These new includes fix them.